elementor.com.website.json

```diff
@@ -0,0 +1,31 @@
+{  
+   "providerId":"elementor.com",
+   "providerName":"Elementor",
+   "serviceId":"website",
+   "serviceName":"Elementor Website",
+   "version": 1,
+   "logoUrl":"https://elementor.com/marketing/wp-content/uploads/2021/10/Elementor-Logo-Full-Red.png",
+   "description":"Enables a domain to work with Elementor",
+   "syncPubKeyDomain": "domainconnect.elementor.com",
+   "hostRequired":true,
```

Comment:
This template looks like an APEX template, so I think `hostRequired` = false is what you want.